File: c++/task10.cpp
#include<iostream>
using namespace std;
int main ()
{
    char str[50];
    cout << "Enter string 1 : ";
    cin>>str;
for(int i=0;str[i]!='\0';i++)
{

if(str[i]>='A'&& str[i]<='Z')
str[i]=str[i]+32;
}
cout<<str;
}

File: java/task10.java
import java.util.Scanner;
import java.lang.Math;
public class task10
{
    
public static void main(String[] args) {
    
Scanner sc=new Scanner(System.in);
System.out.println("enter the limit");
int n=sc.nextInt();

for(int i=1;i<=n;i++)
System.out.println("Number is : "+i+"and cube of 1 is : "+Math.pow(i,3));
}
}